Libraries/Lists/FlatList.js

```diff
@@ -325,7 +325,9 @@ class FlatList<ItemT> extends React.PureComponent<DefaultProps, Props<ItemT>, vo
   }
 
   /**
-   * Scroll to a specific content pixel offset, like a normal `ScrollView`.
+   * Scroll to a specific content pixel offset in the list.
+   * 
+   * Check out [scrollToOffset](virtualizedlist.html#scrolltooffset) of VirtualizedList
```

Comment:
Can you link to "docs/virtualizedlist.html#scrolltooffset" instead? This will ensure the link works regardless of what version of the docs the user is browsing.